src/main/java/cz/smarteon/loxone/LoxoneEndpoint.java

```diff
@@ -15,17 +15,34 @@
 public final class LoxoneEndpoint {
 
     private final String address;
-    private final int port;
+    private int port;
     private final boolean useSsl;
+    private boolean isCloud;
```

Comment:
the naming does not make sense - it must not be cloud address, it's just adress with path